sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/batch_starter.py

```diff
@@ -525,17 +526,142 @@ def generate_queue_url(event):
     return queue_url
 
 
+def get_latest_repoint_file_from_db(session):
+    """Retrieve the latest version of repoint file.
+
+    Parameters
+    ----------
+    session : sqlalchemy.orm.Session
+        Active SQLAlchemy session for querying the database.
+
+    Returns
+    -------
+    str
+        The file name for the latest repoint file.
+    """
+    # Look for latest and greatest file_path in the repointing table.
+    query = (
+        session.query(models.RepointTable)
+        .order_by(desc(models.RepointTable.file_path))
+        .limit(1)
+    )
+    latest_repoint_file = query.first()
```

Comment:
The limit(1) and first() are redundant and can be combined.
```suggestion
    latest_repoint_file = (
        session.query(models.RepointTable)
        .order_by(desc(models.RepointTable.file_path))
        .first()
    )
```